src/internal/connector/exchange/service_iterators.go

```diff
@@ -185,6 +190,13 @@ func filterContainersAndFillCollections(
 		graph.NewMetadataEntry(graph.PreviousPathFileName, currPaths),
 	}
 
+	logger.Ctx(ctx).Infow(
+		"persisted metadata",
```

Comment:
nit: since we haven't done any persistence yet.
```suggestion
		"adding metadata collections",
```